src/app/public/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.scss

```diff
@@ -0,0 +1,130 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+@import "../scss/progress-indicator-variables";
+
+:host {
+  display: flex;
+  flex: 1 0 auto;
```

Comment:
The status marker component only represents a single "marker", so if they're used in a group (i.e., the horizontal status markers `div`) then the host element will need to be set to `flex`.

Reply:
Hmm... children of a flex container don't need `display:flex` to enable a flex context. Only the parent needs it. Maybe I'm missing something?

Reply:
Why do we need flex container properties on the host if the host's only child (`.sky-progress-indicator-status-marker`) also has flex container properties? Is this really necessary?

Reply:
The style was added to allow the status marker to stretch the full height of the item's content (in vertical mode). I'll adjust the styles so it's easier to understand.